### PR TITLE
feat: add repeat purchase header to differentiate correctly between actions

### PR DESCRIPTION
**Pull request by Nisha1306**
Ref: https://github.com/AtB-AS/kundevendt/issues/1666

Before: 

image

After:

image (1)

Figma Link: https://www.figma.com/file/zdZwvobgpEWSagKt0tderx/App-1.2?node-id=6648%3A45714

### CODE REVIEW

**Comment by gorandalum** [comment]
Could you add before/after pictures in this PR? It's nice for small changes like this, then we don't need to run it locally to look at the changes.

File: src/screens/Ticketing/Tickets/RecentTickets/RecentTickets.tsx [hunk 1]
```diff
@@ -64,6 +64,15 @@ export const RecentTickets = () => {
     <View
       style={{backgroundColor: theme.static.background.background_1.background}}
     >
+      <ThemeText
+        type="body__secondary"
+        style={{
+          marginTop: theme.spacings.xLarge,
+          marginLeft: theme.spacings.xLarge,
+        }}
```
Comment (gorandalum) [inline src/screens/Ticketing/Tickets/RecentTickets/RecentTickets.tsx:72]: Set these fields in `useStyles` hook instead. I see there are other places where this hasn't been followed in this file, but I think it is good to do it "correctly" even though.

**Comment by Nisha1306** [comment]
@gorandalum  done..could you please review again.

File: src/screens/Ticketing/Tickets/RecentTickets/RecentTickets.tsx [hunk 2]
```diff
@@ -64,6 +64,9 @@ export const RecentTickets = () => {
     <View
       style={{backgroundColor: theme.static.background.background_1.background}}
     >
+      <ThemeText type="body__secondary" style={styles.header}>
+        {t(RecentTicketsTexts.repeatPurchase.label)}
+      </ThemeText>
```
Comment (gorandalum) [inline src/screens/Ticketing/Tickets/RecentTickets/RecentTickets.tsx:69]: Now this heading is shown also when there is no recent purchases, which will probably look a little odd. Maybe move it below into the `!loading && !!recentTickets.length` check?

**Review by gorandalum — CHANGES_REQUESTED** [review]

File: src/screens/Ticketing/Tickets/RecentTickets/RecentTickets.tsx (hunk 2 above)
Comment (Nisha1306) [reply src/screens/Ticketing/Tickets/RecentTickets/RecentTickets.tsx]: Oopsie..ofcourse :) Fixed.

**Review by gorandalum — APPROVED** [review]

**Pull request merged**